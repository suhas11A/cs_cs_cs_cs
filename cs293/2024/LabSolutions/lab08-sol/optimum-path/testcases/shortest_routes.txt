town_hall main_circle hospital school park offices mall
0 1
0 5
1 3
2 6
3 4
3 2
5 4
3 6
6 5