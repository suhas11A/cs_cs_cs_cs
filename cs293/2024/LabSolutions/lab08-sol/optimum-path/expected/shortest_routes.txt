0
0 1
0 1 3 2
0 1 3
0 5 4
0 5
0 1 3 6